### ISSUE TITLE
Has the Baker screenshots folder moved since Lion 10.7.5? 

### ISSUE THREAD

**Issue by CatherineEng** [Simbul/baker#527]
Hi - I can't seem to find this screenshots folder in the Library folder anymore: https://github.com/Simbul/baker/wiki/Take-pages-screenshots-for-a-Baker-book and, in the debug console, there is no path to a screenshots directory that I can see - but instead this message: 

```

available orientation: both
2012-11-02 03:24:50.038 MyApp[8509:15203] rendering type: three-cards
2012-11-02 03:24:50.039 MyApp[8509:15203] • Reset page details array and empty screenshot directory

```

Any ideas I could try? Thanks in advance.

**Comment by folletto** [comment]
It's not really something under our control, Apple might have changed something with the latest versions of Xcode, which version are you using so we can check it? :)

**Comment by CatherineEng** [comment]
Hi Davide,

How are you? Thanks for replying. 

I'm using Xcode 4.3.3. / Lion 10.7.5.

I suppose if all else fails I could simply manually create the screenshots, naming them using the same conventions and doing a set for both orientations...? Three-cards works so nicely and without the screenshots in my latest build it feels like something is missing.

Best,
Catherine

**Comment by folletto** [comment]
Yes, you can indeed create manually the screenshots... it's just a bit slower, but I guess that if you do them at once on the device and transfer them it's still quite quick. :)

**Comment by CatherineEng** [comment]
Hi - in the logs I get this message after I startup in the simulator. Does that mean the issue is not using screenshots? They are in the directory.

```
• Reset page details array and empty screenshot directory
```

**Comment by Simbul** [comment]
Hi @CatherineEng.

It looks like the issue is that you're not actually using the `screenshots` mode at all. Your log should say:

```
2012-11-02 03:24:50.038 MyApp[8509:15203] rendering type: screenshots
```

instead of

```
2012-11-02 03:24:50.038 MyApp[8509:15203] rendering type: three-cards
```

Sorry for the obvious question, but have you made sure the `-baker-rendering` is correctly set to `screenshots` in your `book.json`? :)

**Comment by CatherineEng** [comment]
Hello Alessandro - thanks. That does indeed help things :-D Silly oversight on my part. Once I had captured the screens I switched it to three-cards to make use of the caching.. then duplicated the project and never changed that value back in the new book.json file. 

One thing I noticed is that if the parameter `"-baker-page-screenshots": "images/screenshots/"` is still in the book.json file, even though the parameter `"-baker-rendering": screenshots` is set and the folder is created for the screenshots - none of them appear to save from the simulator. The application must think there are already screenshots - so that line has to be temporarily removed to do the screenshots as well. 

This is probably not something most people would have to worry about, only if you were doing multiple issues and iterating on top of each one rather than starting from an original book.json file. 

Thanks again!! You saved me a ton of time.

**Comment by Simbul** [comment]
Yes, that's how it's supposed to work: if you provide a screenshots path, screenshots won't be automatically generated (since Baker can't be sure the provided path is writable).
It's in a footnote in the [wiki page](https://github.com/Simbul/baker/wiki/Take-pages-screenshots-for-a-Baker-book), but I'll reword it to make it a bit more clear.

I'll also close this issue if that's ok with you :)

**Comment by CatherineEng** [comment]
Yes, it's very logical that the application wouldn't waste resources saving screenshots again with this parameter present. This issue is resolved for me. 

Thanks,
Catheirne
